### PR TITLE
Doc: add PIV worker and update contributors doc

### PR DESCRIPTION
**Pull request by pbougue**
after #323

### CODE REVIEW

File: documentation/internal_format.md [hunk 1]
```diff
@@ -23,8 +23,15 @@ id | String, Required | Unique code of the contributor. It must be known by Krak
 navitia_coverage | String, Required | Navitia coverage to be used while calling navitia to retrieve corresponding element
 navitia_token | String, Optional | Navitia token to be used while calling navitia to retrieve corresponding element
 feed_url | String, Optional | Url to retrieve the realtime information feed (for polled sources)
+retrieval_interval | Integer, Optional | Minimal interval to retrieve the realtime information feed (for polled sources)
 connector_type | Enum, Required | Type of connector (possible values are `cots`, `gtfs-rt`)
 is_active | Boolean, Optional | Used to activate/deactivate the kirin service for the contributor (default value `true`)
+broker_url| Integer, Optional | Url of the AMQP broker to listen for realtime information queue (case of `piv` contributor only for now)
+exchange_name| Integer, Optional | Exchange of the AMQP broker to listen for realtime information queue (case of `piv` contributor only for now)
+queue_name| Integer, Optional | Queue of the AMQP broker to listen for realtime information queue (case of `piv` contributor only for now)
```
Comment (woshilapin) [inline documentation/internal_format.md:31]:
```suggestion
broker_url| String, Optional | Url of the AMQP broker to listen for realtime information queue (case of `piv` contributor only for now)
exchange_name| String, Optional | Exchange of the AMQP broker to listen for realtime information queue (case of `piv` contributor only for now)
queue_name| String, Optional | Queue of the AMQP broker to listen for realtime information queue (case of `piv` contributor only for now)
```

Comment (woshilapin) [inline documentation/internal_format.md:31]: Maybe add an information about the fact that this queue is bounded to the exchange described above?
```suggestion
broker_url| Integer, Optional | Url of the AMQP broker to listen for realtime information queue (case of `piv` contributor only for now)
exchange_name| Integer, Optional | Exchange of the AMQP broker to listen for realtime information queue (case of `piv` contributor only for now)
queue_name| Integer, Optional | Queue of the AMQP broker, bounded to the above Exchange, to listen for realtime information queue (case of `piv` contributor only for now)
```

File: documentation/internal_format.md [hunk 2]
```diff
@@ -23,8 +23,15 @@ id | String, Required | Unique code of the contributor. It must be known by Krak
...
+
+Note: `broker_url`, `exchange_name` and `queue_name` are used for `piv` contributor only for now.
```
Comment (woshilapin) [inline documentation/internal_format.md:33]: If you have this **note**, maybe you do not need the

> (case of `piv` contributor only for now)

on all of the descriptions above?

File: readme.md [hunk 3]
```diff
@@ -183,20 +183,40 @@ curl -X GET 'http://localhost:5000/contributors'
...
       "is_active": true,
       "navitia_token": "9489dd5f-46b4hhhhhhhhhhhhhhhh",
       "connector_type": "cots",
-      "id": "realtime.cots"
+      "id": "realtime.cots",
+      "feed_url": "*not_used_for_cots*",
+      "retrieval_interval": "*not_used_for_cots*",
+      "broker_url": "*not_used_for_cots*",
+      "exchange_name": "*not_used_for_cots*",
+      "queue_name": "*not_used_for_cots*"
```
Comment (woshilapin) [inline readme.md:195]: Nice way of documenting it :smile:

**Review by woshilapin — APPROVED** [review]

File: documentation/internal_format.md (hunk 1 above)
Comment (pbougue) [reply documentation/internal_format.md:31]: Yes, I didn't know if I should document all those "exchange need to be created before, etc." and if so, where (probably more in main README, then)  ?

File: readme.md (hunk 3 above)
Comment (pbougue) [reply readme.md:195]: Not so fan of it, the only alternative I can think of is making a separate description of each type of connector in the doc.

File: documentation/internal_format.md (hunk 1 above)
Comment (woshilapin) [reply documentation/internal_format.md]: Maybe in the section where you explain about the PivWorker, telling that this component needs an existing `Exchange` to work properly.

File: readme.md (hunk 3 above)
Comment (woshilapin) [reply readme.md:195]: Maybe for later, I'm fine with this for now at least.

**Review by woshilapin — APPROVED** [review]

File: readme.md (hunk 3 above)
Comment (pbougue) [reply readme.md:195]: I specified roles in https://github.com/CanalTP/kirin/pull/330/commits/264e41aa6c50c67a4d475685e0cf96f8991028b0 , it should help on that point.

File: documentation/internal_format.md (hunk 1 above)
Comment (pbougue) [reply documentation/internal_format.md]: Done in the "role" section of the contributing.md: https://github.com/CanalTP/kirin/pull/330/commits/264e41aa6c50c67a4d475685e0cf96f8991028b0

**Review by woshilapin — APPROVED** [review]

**Comment by patochectp** [comment]
in `contributing.md` in section `pgAdmin` we said that user and password are default kirin / kirin, but in `readme.md` in `Setup the Kirin database` we setup database with navitia / navitia.

**Comment by pbougue** [comment]
@patochectp This was confusing indeed.
Removed the "wrong" defaults, so navitia / navitia is the only default.

**Review by patochectp — APPROVED** [review]

**Pull request merged**